pkgs/tools/misc/termplay/default.nix

```diff
@@ -1,18 +1,19 @@
 { rustPlatform, fetchFromGitHub, lib, makeWrapper, gst_all_1, libsixel }:
+
 rustPlatform.buildRustPackage rec {
   pname = "termplay";
-  version = "2.0.4";
+  version = "2.0.6";
```

Comment:
🙋‍♂ I appears the version name is invalid, it should be `v2.0.6` (source:
https://github.com/jD91mZM2/termplay/releases/tag/v2.0.6)

Reply:
Ok, the fact that this works on my machine scares me

Reply:
Ah, before publishing the tag I tried it once with direct revision. Since then the hash hasn't changed, of course, since a tag is just a nicer name for a revision. And so it didn't get re-evaluated when I changed the tag.